cmd/agent/checkers.go

```diff
@@ -98,9 +98,9 @@ func addToMaster(node agent.Agent, config *config, kubeConfig monitoring.KubeCon
 	}
 
 	timeDriftHealth, err := monitoring.TimeDriftHealth(monitoring.TimeDriftCheckerConfig{
-		CAFile:     config.agentCAFile,
-		CertFile:   config.agentCertFile,
-		KeyFile:    config.agentKeyFile,
+		CAFile:     node.GetConfig().CAFile,
```

Comment:
Just curious, what was wrong with how these fields were set before?

Reply:
https://github.com/gravitational/satellite/blob/38ad0d30a6f68c28125b3c56a92137681f282c26/cmd/agent/main.go#L149

In cmd/agent/main.go, the `agentCAFile`, `agentCertFile`, and `agentKeyFile` are left out of the `monitorConf` variable. So the current code was not using any of those credentials when initializing the time drift checker.

We could instead just add the missing fields to `monitorConf`, but this was how planet implemented this portion of the code, so I thought it'd be best to keep it consistent.